pypeit/images/rawimage.py

```diff
@@ -226,7 +226,7 @@ def process(self, par, bpm=bpm, flatimages=None, bias=None,
                 msgs.error("Need to provide slits to create illumination flat")
             illum_flat = flatimages.fit2illumflat(slits, flexure_shift=self.spat_flexure_shift)
             if debug:
-                from pypeit import ginga
+                from pypeit.ginga import ginga
```

Comment:
Please put this at the top of the module.

Reply:
Done, although this was not my change.